src/lib/Menu/ContentRightSidebarBuilder.php

```diff
@@ -225,13 +236,9 @@ public function createStructure(array $options): ItemInterface
                 self::ITEM__MOVE,
                 [
                     'extras' => ['icon' => 'move'],
-                    'attributes' => [
-                        'class' => 'btn--udw-move',
-                        'data-udw-config' => $this->udwExtension->renderUniversalDiscoveryWidgetConfig('single_container'),
-                        'data-root-location' => $this->configResolver->getParameter(
-                            'universal_discovery_widget_module.default_location_id'
-                        ),
-                    ],
+                    'attributes' => $hasCreatePermission
```

Comment:
hmm, you don't need location permissions to move? Just asking, because it stands out, but maybe in this context it works differently.

Reply:
to move you only need content read and create permissions, I tested it.
For a copy, you need also permission to manage location